src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Do we need this file?

Reply:
Yes

Reply:
Then this file should probably contain more than a commented line? :-)